### ISSUE TITLE
Expose shortcuts for the extension using the commands api

### ISSUE THREAD

**Issue by phiter** [vuejs/vue-devtools#957]
### What problem does this feature solve?
I use an extension called Dark Reader that allows me to set up a shortcut for some of its actions. I think this is a nice functionality that Vue Devtools could have. 
There are many commands that could be exposed, for example:

- Inspect Vue Element
- Open in editor (current component)
- Inspect DOM (current component)

These shortcuts can be configured trough chrome://extensions/shortcuts

### What does the proposed API look like?
This article explains how to set it up:

https://developer.chrome.com/apps/commands

So, basically, we'd have the possibility to trigger vue devtools actionsby using shortcuts, like we can do with normal HTML elements pressing CTRL SHIFT C.

We could even have some defaults, like CTRL SHIFT V to inspect a Vue component ;)
